docs/internal-documentation/maintenance/Instructions-for-translators.md

```diff
@@ -0,0 +1,405 @@
+  ```
+
+* Enter the directory of the clone created above or already existing clone:
+  ```
+  cd zonemaster-engine
+  ```
+
+* If you already have an old clone of Zonemaster-Engine, run an update.
+  ```
+  git fetch --all
+  ```
+
+* Now it is time to connect your own fork of *Zonemaster-Engine* at Github to
+  the created clone, unless you have alreday done that, in case you can skip the
+  next step.
+
+* You have a user name at Github. Here we use "xxxx" as your user name and also
+  the name of the remote in clone on the local machine.
+  ```
+  git remote add xxxx git@github.com:xxxx/zonemaster-engine.git
+  git fetch --all
+  ```
+
+Now you should have updated local clones of all three repositories.
+
+
+## Translation steps
+
+The steps in this section will cover most translation work. The steps are the
+same for all three repositories.
+
+* Check-out the *develop* branch and create a new branch to work in. You can call
+  the new branch whatever you want, but here we use the name
+  "translation-update". If that name is already taken, you have to give it a new
+  name or remove the old branch.
+  ```
+  git checkout origin/develop
+  git checkout -b translation-update
+  ```
+
+* Go to the *share* directory and run the update command for the PO file for
+  the language you are going to work with. Replace "xx" with the language code in
+  question. This should be done every time.
+  ```
+  cd share
+  ./update-po xx.po
+  ```
+
+* The PO file is updated with new *msgids*, if any, and now you can start
+  working with it. Unless this is the first translation you only have to work
+  with updated or untranslated items.
+
+* Update the PO file with the tool of your choice. See above. You can copy
+  the PO file to another computer, edit it there, and then copy it back to
+  your Zonemaster-Engine clone -- or Zonemaster-CLI clone or Zonemaster-Backend
+  clone depending on what repository you do the translation for.
+
+* When doing the update, do not change the *msgid*, only the *msgstr*. The
+  *msgid* cannot be be update in this process. They are the links between the
+  Perl module and the PO file.
+
+  * If you find a *msgid* that needs an update please see section
+    [Updates to msgids] for how to report it.
+
+* Inspect every *fuzzy entry* (tagged with "fuzzy"). Update *msgstr*
+  if needed and remove the "fuzzy" tag. The "fuzzy" tag must always be removed.
+
+* Search for *untranslated entries* (empty *msgstr*) and add a
+  translation.
+
+  * At the end of the file there could be *obsolete entries* (lines starting with
+    "#~") and those could have matching translations, especially of the *msgid*
+    has been changed.
+
+* Any remaining *obsolete entries* (lines at the end of the file starting
+  with "#~") could be removed. They serve no purpose anymore.
+
+* *This step is for Zonemaster-Engine only and will only work there.* With the
+  following command, check that the messages arguments (`{...}`) in all *msgstr*
```

Comment:
Maybe remove `With the following command`?

Reply:
And it could remain.